src/Common.jl

```diff
@@ -330,6 +335,39 @@ function Chen2022_exponential_pdf(a::FT, b::FT, c::FT, λ_inv::FT, k::Int) where
     return a * exp(δ * log(1 / λ_inv) - (b + δ) * log(1 / λ_inv + c)) * SF.gamma(b + δ) / SF.gamma(δ)
 end
 
+"""
+    liquid_particle_terminal_velocity(velocity, ρₐ)
+    liquid_particle_terminal_velocity(velocity, ρₐ, D)
+
+Compute the terminal velocity of a liquid particle as a function of its size 
+    (maximum dimension, `D`) using the Chen 2022 parametrization.
+
+# Arguments
+- `velocity`: a struct with terminal velocity parameters from Chen 2022
+- `ρₐ`: air density [kg/m³]
+- `D`: maximum particle dimension [m]
+
+# Returns
+- The first method returns a function of `D`, while the second evaluates at a specific `D`.
+
+Needed for numerical integrals in the P3 scheme.
+
+!!! note
+    We use the same terminal velocity parametrization for cloud and rain water.
+"""
+function liquid_particle_terminal_velocity(velocity::CMP.Chen2022VelTypeRain, ρₐ)
+    (ai, bi, ci) = Chen2022_vel_coeffs_B1(velocity, ρₐ)
+    v_term(D) = sum(@. sum(ai * D^bi * exp(-ci * D)))
```

Comment:
Will this compile on the GPUs in Atmos? I remember fixing something similar here:
https://github.com/CliMA/CloudMicrophysics.jl/pull/545/files

Reply:
Here is the issue for it: https://github.com/CliMA/CloudMicrophysics.jl/issues/544

Reply:
We should add some tests for this, somehow.

Reply:
Follow pattern from here:
https://github.com/CliMA/CloudMicrophysics.jl/blob/main/test/gpu_clima_core_test.jl#L97-L105